api/v1/trigger_types.go

```diff
@@ -26,22 +26,28 @@ import (
 type TriggerSpec struct {
 	// OperationSet is the name of referenced operation set in the generated diagnosis.
 	OperationSet string `json:"operationSet"`
+	// NodeName is the default node which the diagnosis is on.
+	// +optional
+	NodeName string `json:"nodeName,omitempty"`
```

Comment:
`NodeName` can be an array, so scheduled tasks can be uniformly dispatched to multiple nodes.

Reply:
This involves too much change to the API. It should be discussed in a separated issue and submitted in another pull request.